app.rambox.Rambox.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: app.rambox.Rambox
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+rename-desktop-file: rambox.desktop
+rename-icon: rambox
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
Yes. There issues on their GitHub about Wayland bugs that got fixed.